<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EntryExporter</class>
 <widget class="QDialog" name="EntryExporter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>426</width>
    <height>349</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>11</y>
     <width>411</width>
     <height>331</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0" colspan="2">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="ExportEntry">
       <attribute name="icon">
        <iconset resource="images.qrc">
         <normaloff>:/icons/page_white.png</normaloff>:/icons/page_white.png</iconset>
       </attribute>
       <attribute name="title">
        <string>Export &amp;Single</string>
       </attribute>
       <attribute name="toolTip">
        <string>Export single (currently selected) entry</string>
       </attribute>
       <widget class="QWidget" name="layoutWidget1">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>391</width>
          <height>181</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,1" columnstretch="1,1">
         <item row="0" column="0" colspan="2">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Filename</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QLineEdit" name="FileName">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QGroupBox" name="groupBox_2">
           <property name="title">
            <string>Export Format</string>
           </property>
           <widget class="QWidget" name="layoutWidget2">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>20</y>
              <width>171</width>
              <height>101</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QRadioButton" name="HTML">
               <property name="toolTip">
                <string>&lt;p&gt;Export the entry to a HTML-formatted document that may be viewed with a web browser. This option preserves text formatting. &lt;/p&gt;</string>
               </property>
               <property name="text">
                <string>HTML (recommended)</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="PlainText">
               <property name="toolTip">
                <string>&lt;p&gt;Export the entry to a plain text document that may be viewed with a text editor. This option &lt;b&gt;does not&lt;/b&gt; preserve text formatting.&lt;/p&gt;</string>
               </property>
               <property name="text">
                <string>Plain Text</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_3">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::MinimumExpanding</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>Filename Options</string>
           </property>
           <widget class="QWidget" name="layoutWidget3">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>20</y>
              <width>171</width>
              <height>101</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QCheckBox" name="IncludeDate">
               <property name="toolTip">
                <string>&lt;p&gt;Include the date on which the entry was made in the filename. (recommended)&lt;/p&gt;</string>
               </property>
               <property name="text">
                <string>Include Date </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="IncludeJournalName">
               <property name="toolTip">
                <string>&lt;p&gt;Include the name of the journal this entry came from in the filename. (recommended)&lt;/p&gt;</string>
               </property>
               <property name="text">
                <string>Include Journal Name</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_4">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::MinimumExpanding</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="ExportDatabase">
       <attribute name="icon">
        <iconset resource="images.qrc">
         <normaloff>:/icons/page_white_stack.png</normaloff>:/icons/page_white_stack.png</iconset>
       </attribute>
       <attribute name="title">
        <string>Export &amp;All</string>
       </attribute>
       <attribute name="toolTip">
        <string>Export the entire database to one file</string>
       </attribute>
       <widget class="QWidget" name="layoutWidget8">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>391</width>
          <height>181</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_3" rowstretch="0,0,1">
         <item row="0" column="0" colspan="2">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Filename</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QLineEdit" name="FileName_2">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QGroupBox" name="groupBox_3">
           <property name="title">
            <string>Export Format</string>
           </property>
           <widget class="QWidget" name="layoutWidget_2">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>20</y>
              <width>171</width>
              <height>101</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QRadioButton" name="HTML_2">
               <property name="toolTip">
                <string>&lt;p&gt;Export the entry to a HTML-formatted document that may be viewed with a web browser. This option preserves text formatting. &lt;/p&gt;</string>
               </property>
               <property name="text">
                <string>HTML (recommended)</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="PlainText_2">
               <property name="toolTip">
                <string>&lt;p&gt;Export the entry to a plain text document that may be viewed with a text editor. This option &lt;b&gt;does not&lt;/b&gt; preserve text formatting.&lt;/p&gt;</string>
               </property>
               <property name="text">
                <string>Plain Text</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_5">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::MinimumExpanding</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QGroupBox" name="groupBox_4">
           <property name="title">
            <string>File Options</string>
           </property>
           <widget class="QWidget" name="layoutWidget6">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>20</y>
              <width>171</width>
              <height>101</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QCheckBox" name="IncludeExportDate">
               <property name="text">
                <string>Include Export Date</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="asc">
               <property name="toolTip">
                <string>Place oldest entry first</string>
               </property>
               <property name="text">
                <string>Sort Ascending</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="desc">
               <property name="toolTip">
                <string>Place newest entry first</string>
               </property>
               <property name="text">
                <string>Sort Descending</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_2">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::MinimumExpanding</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>41</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Export Directory</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLineEdit" name="ExportLocation"/>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="BrowseButton">
      <property name="text">
       <string>&amp;Browse</string>
      </property>
      <property name="icon">
       <iconset resource="images.qrc">
        <normaloff>:/icons/folder.png</normaloff>:/icons/folder.png</iconset>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="2">
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Maximum</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>6</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>EntryExporter</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>EntryExporter</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
